### ISSUE TITLE
New term proposal : isotopolog_dose

### ISSUE THREAD

**Issue by mslarae13** [GenomicsStandardsConsortium/mixs#586]
**New term details**
For us to assess a new term request we require the following details:
```
Term name - Dose of isotopologue 
Structured comment name - isotopolog_dose
Definition - [a clear and concise description of the term including ]
Expected value - [e.g. text or EFO and/or OBI etc...]
Value syntax - [e.g. {float} {unit}|{termLabel} {[termID]}|{text}|{timestamp} etc...]
Example - [provide and example value]
Preferred unit - [if appropriate]
Package(s) - new checklist term applicable to all packages / extensions
```

**Additional context**
Add any other context about the new term here.

**Comment by simpso91** [comment]
Term name - Dose of isotopologue 
Structured comment name - isotopolog_dose
Definition - Total dose of isotopologue added to the system in grams
Expected value - value
Value syntax - {float} {unit}
Example - 0.1 g or   0.1 g | 0.3 g
Preferred unit - grams

**Comment by mslarae13** [comment]
@simpso91  can this be multivalued? does order matter (considering https://github.com/GenomicsStandardsConsortium/mixs/issues/577 )

**Comment by simpso91** [comment]
Yes, it can be multivalued and the order should be the same as isotopologue (#577)!

Sorry, things with "x | y" in the examples can be multivalued. I should have been clearer about this in the definition of each term.

**Comment by simpso91** [comment]
We have decided to use "ppm" as the preferred unit instead of "g/mL" to more flexibly accept solid, liquid, and gas environments.
